solr/core/src/test/org/apache/solr/handler/component/TestPivotHelperCode.java

```diff
@@ -89,8 +89,7 @@ public void testCompareWithNullLast() {
     assertEquals(a.compareTo(b), PivotFacetFieldValueCollection.compareWithNullLast(a, b));
     assertEquals(b.compareTo(a), PivotFacetFieldValueCollection.compareWithNullLast(b, a));
 
-    Long bb = new Long(b.longValue());
-    assertEquals(0, PivotFacetFieldValueCollection.compareWithNullLast(b, bb));
+    assertEquals(0, PivotFacetFieldValueCollection.compareWithNullLast(b, b));
```

Comment:
hmm, not sure about this one, looks like it's part of the test?

Reply:
As I edited it, I wondered if you were going to call this one out!  Eagle Eyes!    I could totally be swayed to undo it.     The line `Long bb = new Long(b.longValue()); ` means we are basically testing Java functionality???

Reply:
you know what, I'll just roll this cahnge back ;-)